task/3scale-toolbox-import/0.1/tests/run.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: pipeline-import-openapi-sample
+spec:
+  tasks:
+    - name: 3scale-toolbox-import
+      params:
+        - name: CONTEXT_DIR
+          value: shared-workspace
+        - name: PATH_OPENAPI
+          value: https://raw.githubusercontent.com/rafamqrs/camel-quarkus-openapi/main/swagger.json
+        - name: PRIVATE_BASE_URL
+          value: http://sample.svc.cluster.local:8080
+        - name: TARGET_SYSTEM_NAME
+          value: sample_api
+        - name: DESTINATION
+          value: https://13ec5907fc051adf4ee4b419882c0f55d6b000c8b26495081551933402a76906@3scale-admin.apps.cluster-7f5a.7f5a.example.opentlc.com
```

Comment:
is it a short lived cluster or will it be available all the time? Asking because once the PR is merged the task will be tested on a nightly basis and if the cluster goes down then it will result in the task's test failure.

Reply:
it is a short liver cluster, but it will be available until  11/20/21 01:20 UTC. The only that I test this integration is using a 3scale instance running, without it, I can't test.

Reply:
how about you mock the call? You can take the example of [github-add-comment task tests](https://github.com/tektoncd/catalog/tree/main/task/github-add-comment/0.5/tests)

Reply:
It is a little bit hard to mock the endpoint, let me try to explain:
I'm using the 3scale toolbox cli from an image, it was built using the ruby, and basically it abstract all the interaction with the 3scale API, so to mock the calls, I will have to open the 3scale toolbox code and identify all request to 3scale API, due to there are many endpoints the toolbox called before saving the API. 

I can keep the cluster running maybe for one week if necessary, but if you think is better mock the call, I think is best to use the 3scale api instead of the 3scale toolbox.

Reply:
Hi @vinamra28, do you agree if I continue using the cluster?